default/python/AI/TechsListsAI.py

```diff
@@ -346,11 +340,8 @@ def __init__(self):
             self.economy,
             self.economy,
             Dep.LRN_ARTIF_MINDS_1,
-            self.defense,
```

Comment:
These removed self.defense entries are in TechGroup2SparseA (the others are in TechGroup2SparseB) - both inherit from TechGroup2 which does not contain defense dict, so it is ok to remove the entries.
Also in a sparse setup you do not need planetary defense soon, makes also sense for the AI.